public/pages/Overview/components/GettingStarted/GettingStartedPopup.tsx

```diff
@@ -33,88 +24,86 @@ export const GettingStartedPopup: React.FC<GettingStartedPopupProps> = ({
       {
         title: 'Create security detector',
         children: (
-          <div style={{ marginTop: '-20px' }}>
-            <EuiText>
-              <p>
-                Identify security findings and threats from your log datas with curated detection
-                rules.
-              </p>
-            </EuiText>
-            <EuiSpacer size="s" />
-            <EuiButton
-              fill
-              onClick={() => {
-                dismissPopup();
-                history.push(ROUTES.DETECTORS_CREATE);
-              }}
-            >
-              Create detector
-            </EuiButton>
-          </div>
+          <GetStartedStep
+            title={
+              'Identify security findings and threats from your log datas with curated detection rules.'
```

Comment:
This line seems to be missing some text. The mocks say "Identify security findings and threats from your log datas with detection rules. Additionally, you can set up alerts based on rule conditions."